src/nvim/getchar.c

```diff
@@ -264,16 +266,16 @@ add_buff (
   buf->bh_index = 0;
 
   ssize_t len;
-  if (buf->bh_space >= (int)slen) {
-    len = STRLEN(buf->bh_curr->b_str);
+  if (buf->bh_space >= slen) {
+    len = (ssize_t)STRLEN(buf->bh_curr->b_str);
     STRLCPY(buf->bh_curr->b_str + len, s, slen + 1);
-    buf->bh_space -= slen;
+    buf->bh_space -= (int)slen;
```

Comment:
Yeah it should be an int. I was actually gonna look this over today to make sure that I didn't miss anything. Thanks for pointing that out.